src/main/java/jenkins/plugins/clangscanbuild/publisher/ClangScanBuildPublisherDescriptor.java

```diff
@@ -41,10 +40,7 @@ public String getDisplayName() {
 
 	@Override
 	public boolean isApplicable( @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType ){
-		if( !FreeStyleProject.class.isAssignableFrom( jobType ) ){
-			System.err.println( "Clang scan-build ERROR: Expected FreeStyleProject but was: " + jobType + " at Publisher Descriptor" );
-		}
-		return AbstractProject.class.isAssignableFrom( jobType );
+		return true;
```

Comment:
@amuniz ah of course. Duh :)

So the changes to `isApplicable` appear to just simplify/clarify the code, right? And then the other changes are just logging changes, right? If that's the case, then how is this PR fixing JENKINS-24498?